[8.0][WIP] l10n_it_account

dcorio opened the pull request:
l10n_it_account has been created this morning during the sprint because the vat_statement_type is needed by other localization modules which should not depend on account_vat_period_end_statement.
so we moved it to a new module (l10n_it_account).

we're going to use this new module to host other accounting helpers (fields, configurations, defaults, ...)

Commit:
[FIX] Porting vat_registries to 8

Commit:
[FIX] fix import and depends

Commit:
[FIX] Parser mod

Commit:
[FIX] Last 7 Version commit and default change

Commit:
[FIX] fix user and flake8

Commit:
[FIX] Fix method invocation

eLBati committed:
[FIX] metadata

Commit:
[FIX] try to fix qweb

Commit:
[FIX] Add new folder

Commit:
[FIX] fix sales

eLBati committed:
[DEL] mako stuff

eLBati committed:
[IMP] copyright

eLBati committed:
[REF] structure

Commit:
[FIX] Fix style

eLBati committed:
[FIX] line index

eLBati committed:
[ADD] header

eLBati committed:
[FIX] what is entity

eLBati committed:
[FIX] why not page break

eLBati committed:
[FIX] wrong counter

eLBati committed:
[FIX] columns

eLBati committed:
[FIX] CSS

eLBati committed:
[DEL] fiscal_page_base

eLBati committed:
[FIX] header

eLBati committed:
[DEL] mako

eLBati committed:
[IMP] description

eLBati committed:
[FIX] PEP8

eLBati committed:
[FIX] qweb style

eLBati committed:
Merge remote-tracking branch 'origin/8.0' into fix_porting_l10n_it_vat_registries

Conflicts:
	l10n_it_vat_registries/__openerp__.py

eLBati committed:
[FIX] vat registry report not following language

eLBati committed:
[FIX] wrong class name

eLBati committed:
Merge remote-tracking branch 'origin/8.0' into fix_porting_l10n_it_vat_registries

tafaRU committed:
Update .travis file according to https://github.com/OCA/maintainer-quality-tools/pull/186

tafaRU committed:
Add oca_dependencies file

yvaucher committed:
Add bug tracker link on README.rst

tafaRU committed:
Use transifex

tafaRU committed:
Add Translation Status to project

tafaRU committed:
Remove requests, as is now part of travis_install_nightly installation

tafaRU committed:
Merge pull request #111 from tafaRU/8.0-update_travis

Update travis for branch 8.0

tafaRU committed:
Use svg badge for coveralls according to https://github.com/OCA/maintainer-quality-tools/pull/196

pedrobaeza committed:
Merge pull request #113 from tafaRU/8.0-svg-coveralls

Use svg badge for coveralls

yvaucher committed:
Merge pull request #112 from yvaucher/8.0-add-bug-tracker-link

Add bug tracker link on README.rst

eLBati committed:
Merge remote-tracking branch 'origin/8.0' into fix_porting_l10n_it_vat_registries

eLBati committed:
[ADD] style="page-break-inside: avoid" for row content

eLBati committed:
[IMP] using journals to compute tax codes sum
So, the journals selected in print wizard will be used to compute tax code summary

Commit:
[ADD] l10n_it_account

Commit:
[IMP] move vat_statement_type to l10n_it_account

Commit:
[FIX] v8 compat

Commit:
[FIX] typos

tafaRU commented on l10n_it_account/model/account.py:
```diff
@@ -0,0 +1,32 @@
+#
+##############################################################################
+
+from openerp.osv import fields, models
```
Replace `openerp.osv` by `openerp`

Commit:
[IMP] i18n

eLBati commented:
LGTM :+1:

tafaRU commented:
@dcorio is still WIP?

eLBati committed:
[ADD] VAT register: journals can be groups within 1 register and selecting it while printing will load that group of journals

eLBati committed:
[ADD] menu and view for account.tax.register

eLBati committed:
[REF] register -> registry

eLBati committed:
[ADD] translations

Commit:
[IMP] folders/files structure

Commit:
[ADD] common paper formats

Commit:
[FIX] missing view import

dcorio commented:
@tafaRU it is but @eLBati was suggesting to merge it to l10n-italy if it's fine right now and we got the 3 :+1:  of course so we can improve it by creating PR against l10n-italy

Commit:
[FIX] paper orientation

dcorio commented:
@tafaRU still WIP

DAMilani committed:
Merge remote-tracking branch 'abstract-open-solutions/8.0-l10n_it_account' into l10n_it_vat_registries_imp_layout

Commit:
[ADD] undeductable field on account.tax

tafaRU commented on l10n_it_account/__init__.py:
```diff
@@ -0,0 +1,25 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2010 OpenERP Italian Community
```
Please align copyrights to the other headers.

tafaRU commented on l10n_it_account/__openerp__.py:
```diff
@@ -0,0 +1,39 @@
+        'views/account_view.xml',
+        'reports/account_reports_view.xml',
+    ],
+    "qweb": [],
```
Please remove empty keys.

Commit:
[FIX] removed empty keys

tafaRU commented on l10n_it_account/__openerp__.py:
```diff
@@ -0,0 +1,39 @@
+    "qweb": [],
+    "demo": [],
+    "test": [],
+    "active": False,
```
This key is deprecated and it's an alias for `auto_install` whose default value is already `False`. So please remove it.

Commit:
[FIX] removed active key

tafaRU commented:
> @tafaRU still WIP

Ok, but I started to spot some issues concerning OCA guidelines.

DAMilani committed:
Merge pull request #1 from abstract-open-solutions/8.0-l10n_it_account

[ADD] undeductable field on account.tax

dcorio commented:
@tafaRU and i appreciate that :) :+1:

Commit:
[IMP] standard footer

Commit:
[IMP] negative balance helpers for BS report

eLBati committed:
[ADD] porting account_vat_period_end_statement_report from https://github.com/alessandrocamilli/test

eLBati committed:
[IMP] PEP8 and copyright

eLBati committed:
[IMP] translation

DAMilani committed:
Merge pull request #2 from abstract-open-solutions/8.0-l10n_it_account

[IMP] default report layout

DAMilani committed:
Modificata stampa registri Iva
Controllo su univocità conti imposte per le imposte di acquisto

Commit:
[FIX] report title

DAMilani committed:
Merge pull request #4 from abstract-open-solutions/8.0-l10n_it_account

[FIX] report title

DAMilani committed:
Modifica stampa registro iva
Aggiunto dettaglio tasse su riga fattura
Inserita dipendenza da l10n_it_account per gestione header and footer

Commit:
[FIX] terms for non deductible tax

Commit:
[FIX] typos

Commit:
[IMP] i18n

DAMilani committed:
Merge pull request #6 from abstract-open-solutions/8.0-l10n_it_account

[FIX] terms and typos

DAMilani committed:
modifica nome campo Non-deductible in nondeductible

DAMilani committed:
Merge pull request #3 from eLBati/add_vat_statement_report

Add vat statement qweb report

DAMilani committed:
Aggiunto controllo su esistenza conti imponibile e conto iva per calcolo totali

DAMilani committed:
fix calcolo totali iva stampa registri, fix stampa liquidazione controllo tipo conto imposta credit/debit

Commit:
[FIX] terms

dcorio committed:
Merge pull request #5 from linkitspa/l10n_it_vat

L10n it vat
